libafl_qemu/src/lib.rs

```diff
@@ -14,9 +14,14 @@
 #![allow(clippy::borrow_deref_ref)]
 // Allow only ATM, it will be evetually removed
 #![allow(clippy::missing_safety_doc)]
+// libafl_qemu_sys export types with mepty struct markers (e.g. struct {} start_init_save)
+// This causes bindgen to generate empty Rust struct that are generally not FFI-safe due to C++ having empty structs with size 1
+// As the QEMU codebase is C, it is FFI-safe and we just ignore the warning
+#![allow(improper_ctypes)]
```

Comment:
Why not set the allow only for libafl_qemu_sys, instead of the whole crate?

Reply:
Because I'm using types of the sys in FFI code in libafl_qemu.